docpipeline/generate.py

```diff
@@ -86,6 +87,14 @@ def process_blob(blob, credentials, devsite_template):
     blob.download_to_filename(tar_filename)
     log.info(f"Downloaded gs://{blob.bucket.name}/{blob.name} to {tar_filename}")
 
+    # Check to see if api directory exists in the tarball.
+    # If so, only compress things into obj/*
+    tar_file = tarfile.open(tar_filename)
+    for tarinfo in tar_file:
+        if tarinfo.isdir() and "api" in tarinfo.name:
```

Comment:
Should this be `in` or `==`?

Reply:
either works, but we would have a problem if we got a directory with the name containing `api` but not an api directory we're looking for. Thanks for catching that!